WordPress/WordPressApi/WordPressComRestApi.swift

```diff
@@ -244,12 +249,19 @@ final class WordPressComRestAPIResponseSerializer: AFJSONResponseSerializer
     }
 
     override func responseObjectForResponse(response: NSURLResponse?, data: NSData?, error: NSErrorPointer) -> AnyObject? {
-        let responseObject = super.responseObjectForResponse(response, data: data, error: error)
+        var optionalError: NSError?
+        let responseObject = super.responseObjectForResponse(response, data: data, error: &optionalError)
+        error.memory = optionalError
```

Comment:
@koke I'm doing this in order to have a variable with the error and grab the original userInfo from that error, before I add new things to it if needed.